translations/de.json/de.json

```
@@ -0,0 +1,129 @@
+	"Via OSM.org": "Über OSM.org",
+	"Via Mapillary": "Mit Mapillary",
+	"Retour à la page d'accueil": "Zurück zur Startseite",
+	"Projet du mois": "Projekt des Monats",
+	"Carte": "Karte",
+	"Liste": "Liste",
+	"Afficher uniquement les signalements": "Nur Meldungen anzeigen",
+	"Prêt à intégrer": "Bereit zur Integration",
+	"Avec photo": "Mit Foto",
+	"Anonymes": "Anonymisiert",
+	"Aucun signalement pour l'instant !": "Bisher gibt es keine Meldungen.",
+	"commentaire(s)": "Kommentar(e)",
+	"Anonyme": "Anonymisiert",
+	"Ouvert": "Offen",
+	"Fermé": "Geschlossen",
+	"Réouvert": "Wiedereröffnet",
+	"Attributs renseignés ;": "Angegebene Attribute;",
+	"Compléter": "Ergänzen",
+	"Éditer": "Bearbeiten",
+	"Se déconnecter": "Abmelden",
+	"S'inscrire / Se connecter": "Registrieren / Anmelden",
+	"Utilisateur": "Benutzer",
+	"Notifications non-lues": "Nicht gelesene Mitteilungen",
+	"Notifications": "Benachrichtigungen",
+	"Mes badges": "Meine Abzeichen",
+	"Merci !": "Danke!",
+	"Podium": "Podium",
+	"Inscrivez-vous ou connectez-vous pour voir le podium": "Registrieren Sie sich oder melden Sie sich an, um das Podium zu sehen",
+	"Pseudo": "Nickname",
+	"Points": "Punkte",
+	"Avancement": "Fortschritt",
+	"Notes": "Hinweise",
+	"Clés fréquentes": "Häufige Schlüssel",
+	"jours restants": "Noch verbleibende Tage",
+	"jours de contributions": "Anzahl Tage mit Beiträgen",
+	"personnes ayant contribué": "Personen, die einen Beitrag geleistet haben",
+	"En ce moment": "Zur Zeit",
+	"Statistiques": "Statistik",
+	"Données": "Daten",
+	"Retour à l'accueil": "Zurück zur Startseite",
+	"Accueil | Découvrir le projet du mois": "Startseite | Entdecken Sie das Projekt des Monats",
+	"Erreur %s ; rien que le désert ici...": "Fehler %s; hier ist nur Wüste...",
+	"Erreur %s": "Fehler %s",
+	"Contribuer": "Beitrag leisten",
+	"Un grand merci à tous !": "Ein herzliches Dankeschön an alle!",
+	"Merci à toutes les contributrices et tous les contributeurs d'avoir fait de ce projet un succès. Sans vous tous, OpenStreetMap ne serait rien !": "Vielen Dank an alle Mitwirkenden für den Erfolg dieses Projekts. Ohne Euch alle wäre OpenStreetMap nichts!",
+	"Merci": "Danke.",
+	"Voir les statistiques": "Siehe Statistik",
+	"Comment participer": "Wie kann man mitmachen?",
+	"Ce projet du mois est terminé, mais la cartographie dans OpenStreetMap ne s'arrête pas ! Vous pouvez continuer à enrichir les données à l'aide de nos outils et de la documentation.": "Dieses Projekt des Monats ist beendet, aber das Mapping in OpenStreetMap hört nicht auf! Sie können die Daten mit unseren Tools und Dokumentationen weiter anreichern.",
+	"Documentation complète": "Vollständige Dokumentation",
+	"Article de blog": "Blog-Post",
+	"L'ensemble des données produites avant, pendant et après le projet du mois sont disponibles aux adresses suivantes ;": "Alle vor, während und nach dem Projekt des Monats erstellten Daten sind unter den folgenden Adressen verfügbar:",
```

That should be a colon, rather than a semicolon in the attribute name.
```suggestion
	"L'ensemble des données produites avant, pendant et après le projet du mois sont disponibles aux adresses suivantes :": "Alle vor, während und nach dem Projekt des Monats erstellten Daten sind unter den folgenden Adressen verfügbar:",
```